hstream/src/HStream/Server/Handler/Connector.hs

```diff
@@ -38,16 +38,16 @@ createSinkConnectorHandler
   -> ServerRequest 'Normal CreateSinkConnectorRequest Connector
   -> IO (ServerResponse 'Normal Connector)
 createSinkConnectorHandler sc
-  (ServerNormalRequest _ CreateSinkConnectorRequest{..}) = defaultExceptionHandle $ do
-  connector <- createConnector sc (TL.toStrict createSinkConnectorRequestSql)
-  returnResp connector
+  (ServerNormalRequest _ CreateSinkConnectorRequest{..}) = defaultExceptionHandle $
+    createConnector sc (TL.toStrict createSinkConnectorRequestSql)
+      >>= returnResp
```

Comment:
Using `do` is more readable. And if there is more than one expression(in the future), you will need `do`.